### PR TITLE
Switch stats instance to "public-extended" network

### PR DESCRIPTION
**Pull request by kysrpex**
Let the stats instance communicate with the database server.

Should address [this issue](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/1237#issuecomment-2225551700) related to usegalaxy-eu/infrastructure-playbook#1237.

### CODE REVIEW

File: instance_core_stats.tf
Comment (kysrpex) [inline instance_core_stats.tf:11]: Is it ok to switch the networks or do we need to keep the instance connected to both networks?

**Review by mira-miracoli — APPROVED** [review]

**Comment by sanjaysrikakulam** [comment]
Please be advised that changing the [network](https://registry.terraform.io/providers/terraform-provider-openstack/openstack/latest/docs/resources/compute_instance_v2#network) will result in a total recreation of the instance.

**Comment by kysrpex** [comment]
> Please be advised that changing the [network](https://registry.terraform.io/providers/terraform-provider-openstack/openstack/latest/docs/resources/compute_instance_v2#network) will result in a total recreation of the instance.

Thanks a lot for the warning.

:warning: DO NOT MERGE THIS PR unless you are doing the migration described in usegalaxy-eu/infrastructure-playbook#1235, the folder `/data/monitoring` needs to be backed up :exclamation: :exclamation: .

**Comment by mira-miracoli** [comment]
Closing and reopening to make Jenkins run

**Comment by bgruening** [comment]
Can one of the admins verify this patch?

**Pull request merged**
